api/graphql/schema.py

```diff
@@ -1074,6 +1075,31 @@ async def analysis_runner(
             )
         return GraphQLAnalysisRunner.from_internal(analysis_runners[0])
 
+    @strawberry.field
+    async def ourdna_dashboard(self, info: Info, project: GraphQLFilter[str] | None = None) -> list[strawberry.scalars.JSON]:
+        connection = info.context['connection']
+        ptable = ProjectPermissionsTable(connection)
+
+        if not project:
+            raise ValueError('Must provide project')
+
+        project_name_map: dict[str, int] = {}
+        if project:
+            project_names = project.all_values()
+            projects = await ptable.get_and_check_access_to_projects_for_names(
+                user=connection.author, project_names=project_names, readonly=True
+            )
+            project_name_map = {p.name: p.id for p in projects}
+
+        filter_ = SampleFilter(
+            project=(
+                project.to_internal_filter(lambda pname: project_name_map[pname])
+                if project
+                else None
+            ),
+        )
```

Comment:
`project` is always going to have a value here, so you can skip the following nullable checks. Also, do you want this to be multiple projects, or just a single?

Reply:
We were thinking the project here would just be 'OurDNA'. Is there a scenario where another project is valid here?